programs/invariant/invariant-types/src/math.rs

```diff
@@ -1699,86 +1717,91 @@ mod tests {
             assert_eq!(max_price, Price::new(65535383934512647000000000000));
             assert_eq!(
                 calculate_price_sqrt(max_tick),
                 Price::new(65535383934512647000000000000)
             );
 
-            let max_price = get_max_sqrt_price(100);
-            let max_tick: i32 = get_max_tick(100);
+            let max_price = get_max_sqrt_price(100).unwrap();
+            let max_tick: i32 = get_max_tick(100).unwrap();
             assert_eq!(max_price, Price::new(65535383934512647000000000000));
             assert_eq!(
                 calculate_price_sqrt(max_tick),
                 Price::new(65535383934512647000000000000)
             );
         }
         {
-            let min_price = get_min_sqrt_price(1);
-            let min_tick: i32 = get_min_tick(1);
+            let min_price = get_min_sqrt_price(1).unwrap();
+            let min_tick: i32 = get_min_tick(1).unwrap();
             assert_eq!(min_price, Price::new(108822289458000000000000));
             assert_eq!(
                 calculate_price_sqrt(min_tick),
                 Price::new(108822289458000000000000)
             );
 
-            let min_price = get_min_sqrt_price(2);
-            let min_tick: i32 = get_min_tick(2);
+            let min_price = get_min_sqrt_price(2).unwrap();
+            let min_tick: i32 = get_min_tick(2).unwrap();
             assert_eq!(min_price, Price::new(11842290682000000000000));
             assert_eq!(
                 calculate_price_sqrt(min_tick),
                 Price::new(11842290682000000000000)
             );
 
-            let min_price = get_min_sqrt_price(5);
-            let min_tick: i32 = get_min_tick(5);
+            let min_price = get_min_sqrt_price(5).unwrap();
+            let min_tick: i32 = get_min_tick(5).unwrap();
             assert_eq!(min_price, Price::new(15261221000000000000));
             assert_eq!(
                 calculate_price_sqrt(min_tick),
                 Price::new(15261221000000000000)
             );
 
-            let min_price = get_min_sqrt_price(10);
-            let min_tick: i32 = get_min_tick(10);
+            let min_price = get_min_sqrt_price(10).unwrap();
+            let min_tick: i32 = get_min_tick(10).unwrap();
             assert_eq!(min_price, Price::new(15258932000000000000));
             assert_eq!(
                 calculate_price_sqrt(min_tick),
                 Price::new(15258932000000000000)
             );
 
-            let min_price = get_min_sqrt_price(100);
-            let min_tick: i32 = get_min_tick(100);
+            let min_price = get_min_sqrt_price(100).unwrap();
+            let min_tick: i32 = get_min_tick(100).unwrap();
             assert_eq!(min_price, Price::new(15258932000000000000));
             assert_eq!(
                 calculate_price_sqrt(min_tick),
                 Price::new(15258932000000000000)
             );
+
+            get_min_tick(u16::MAX).unwrap_err();
```

Comment:
You can extend tests in some functions to ensure that even with incorrect parameters, the function returns a `Result` instead of panicking. I mean triggering the `Err` spots you added.

For example, `compute_swap_step` with a `fee` that is more than 100%